projects/storefrontlib/src/cms-components/navigation/navigation/navigation-ui.component.html

```diff
@@ -41,6 +41,11 @@ <h5 [attr.aria-label]="node.title">
       </h5>
     </ng-template>
 
+    <span
```

Comment:
1. I would move this logic to `ts` file, it's rather complex
2. Why do we have an empty element here?

Reply:
1. If you think that's complex, you should see the ts file.
2. It works with css without obstructing other functions of this component

Reply:
1. it is a good practice not to have complex logic (`*ngIf="flyout && node.children?.length > 0 && node.url"`) in template file. simple function, `shouldThisAndThatWork()` should be enough
2. smells like a dirty hack to me, but if nothing else works, then leave it as is